modules/ROOT/pages/index.adoc

```diff
@@ -17,15 +17,118 @@ Members of your organization can use Partner Manager to perform a variety of fun
 * Managing the activity of the business transactions with your trading partners
 * Optimizing backend services and performance
 
-== Prerequisites for Using Partner Manager
+== Before You Begin Using Partner Manager
 
-To use Partner Manager, you must be familiar with:
+Before you begin using Partner Manager, you must have:
 
-* EDI message formats
-* How to create DataWeave maps to perform data transformations
+* An Anypoint Platform user account with the Anypoint Partner Manager entitlement
```

Comment:
Do you need Anypoint in front of PM here?

Reply:
Anypoint is in the first paragraph so it is not needed here.